### PR TITLE
feat: initial docs for release practices and project status

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

File: docs/momento-on-github.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+|-----------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ![official](./badges/project-status-official.svg)   | Fully supported projects, such as our official SDKs.  Any bugs, performance issues, or user experience issues in these projects are of utmost importance to us and we will strive to address them as quickly as possible.                                                                                            |
+| ![official](./badges/project-status-incubating.svg) | These projects are considered experimental, not mature enough, or lacking in some manner or release quality such testing and supporting documentation. They are provided as-is. You're welcome to use them.  Projects that garner a lot of community interest will be considered for promotion to 'official' status. |
```
Comment (eaddingtonwhite) [inline docs/momento-on-github.md:26]:
```suggestion
| ![incubating](./badges/project-status-incubating.svg) | These projects are considered experimental, not mature enough, or lacking in release quality such as testing or supporting documentation. They are provided as-is. You're welcome to use them.  Projects that garner a lot of community interest will be considered for promotion to 'official' status. |
```

File: docs/momento-on-github.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Project Stability
+
+| Status                                                   | Description                                                                                                                                                                                                                                                                                                                     |
+|----------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ![official](./badges/project-stability-experimental.svg) | Experimental means "try it only in toy projects".  We are just trying out an idea and want some users to play with it and give feedback. If it doesn't work out, we may drop it any minute.                                                                                                                                     |
```
Comment (eaddingtonwhite) [inline docs/momento-on-github.md:32]:
```suggestion
| ![experimental](./badges/project-stability-experimental.svg) | Experimental means "try it only in toy projects".  We are just trying out an idea and want some users to play with it and give feedback. If it doesn't work out, we may drop it any minute.                                                                                                                                     |
```

File: docs/momento-on-github.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+| ![official](./badges/project-stability-alpha.svg)        | Alpha means "use at your own risk, expect migration issues": We decided to productize this idea, but it hasn't reached the final shape yet.                                                                                                                                                                                     |
```
Comment (eaddingtonwhite) [inline docs/momento-on-github.md:33]:
```suggestion
| ![alpha](./badges/project-stability-alpha.svg)        | Alpha means "use at your own risk, expect migration issues": We decided to productize this idea, but it hasn't reached its final shape yet.                                                                                                                                                                                     |
```

**Review by eaddingtonwhite — APPROVED** [review]
This seems like a great start! Nice @cprice404 few nits but looks awesome 👍

**Review by kvcache — COMMENTED** [review]
Looks good to me aside from the one link label. Thanks

**Review by malandis — APPROVED** [review]

**Review by danielamiao — COMMENTED** [review]
1 comment on badge typo, plus a more general question:

What's the relationship between project status and stability? Are all possible combinations reasonable?
- A project can have `status: official`, and stability `experimental, alpha, beta, or stable`?
- A project can have `status: incubating`, and stability `experimental, alpha, beta, or stable`? Does `stable` really make sense for an incubating project? I can see this being possible but want to clarify.

File: docs/badges/project-stability-alpha.source [hunk 4]
```diff
@@ -0,0 +1 @@
+https://img.shields.io/badge/stabity-alpha-important?colorA=555555&colorB=DB3683
```
[the remark this answers is not in the document]
Comment (cprice404) [reply docs/badges/project-stability-alpha.source:1]: lol dang it

**Comment by cprice404** [comment]
> 1 comment on badge typo, plus a more general question:
> 
> What's the relationship between project status and stability? Are all possible combinations reasonable?
> 
>     * A project can have `status: official`, and stability `experimental, alpha, beta, or stable`?
> 
>     * A project can have `status: incubating`, and stability `experimental, alpha, beta, or stable`? Does `stable` really make sense for an incubating project? I can see this being possible but want to clarify.

I would say that some combinations would be exceedingly rare, especially the incubating+stable combo that you mentioned.  But not impossible, if we had a particularly popular community-driven project that we didn't intend to provide official support for.  So, it doesn't seem worth the effort to try to document any restrictions to the matrix imo?

**Comment by danielamiao** [comment]
> So, it doesn't seem worth the effort to try to document any restrictions to the matrix imo?

Yup, makes sense, I agree, just clarifying for my own understanding.

**Pull request merged**